Binary files libgeos.org/.git/index and libgeos/.git/index differ
diff -burN libgeos.org/.git/logs/HEAD libgeos/.git/logs/HEAD
--- libgeos.org/.git/logs/HEAD	2016-06-14 14:15:23.619557617 +0200
+++ libgeos/.git/logs/HEAD	2016-06-14 09:15:34.507698261 +0200
@@ -1 +1 @@
-0000000000000000000000000000000000000000 714127ef49087634e7f6687424fdf041d18c3d0f Rashad Kanavath <rashad.kanavath@c-s.fr> 1465906523 +0200	clone: from https://github.com/libgeos/libgeos
+0000000000000000000000000000000000000000 714127ef49087634e7f6687424fdf041d18c3d0f Rashad Kanavath <rashad.kanavath@c-s.fr> 1465888534 +0200	clone: from https://github.com/libgeos/libgeos
diff -burN libgeos.org/.git/logs/refs/heads/svn-trunk libgeos/.git/logs/refs/heads/svn-trunk
--- libgeos.org/.git/logs/refs/heads/svn-trunk	2016-06-14 14:15:23.619557617 +0200
+++ libgeos/.git/logs/refs/heads/svn-trunk	2016-06-14 09:15:34.507698261 +0200
@@ -1 +1 @@
-0000000000000000000000000000000000000000 714127ef49087634e7f6687424fdf041d18c3d0f Rashad Kanavath <rashad.kanavath@c-s.fr> 1465906523 +0200	clone: from https://github.com/libgeos/libgeos
+0000000000000000000000000000000000000000 714127ef49087634e7f6687424fdf041d18c3d0f Rashad Kanavath <rashad.kanavath@c-s.fr> 1465888534 +0200	clone: from https://github.com/libgeos/libgeos
diff -burN libgeos.org/.git/logs/refs/remotes/origin/HEAD libgeos/.git/logs/refs/remotes/origin/HEAD
--- libgeos.org/.git/logs/refs/remotes/origin/HEAD	2016-06-14 14:15:23.619557617 +0200
+++ libgeos/.git/logs/refs/remotes/origin/HEAD	2016-06-14 09:15:34.507698261 +0200
@@ -1 +1 @@
-0000000000000000000000000000000000000000 714127ef49087634e7f6687424fdf041d18c3d0f Rashad Kanavath <rashad.kanavath@c-s.fr> 1465906523 +0200	clone: from https://github.com/libgeos/libgeos
+0000000000000000000000000000000000000000 714127ef49087634e7f6687424fdf041d18c3d0f Rashad Kanavath <rashad.kanavath@c-s.fr> 1465888534 +0200	clone: from https://github.com/libgeos/libgeos
diff -burN libgeos.org/src/CMakeLists.txt libgeos/src/CMakeLists.txt
--- libgeos.org/src/CMakeLists.txt	2016-06-14 14:15:23.675557617 +0200
+++ libgeos/src/CMakeLists.txt	2016-06-14 14:14:23.687558086 +0200
@@ -62,7 +62,7 @@
 else()
 
   add_library(geos SHARED ${geos_SOURCES} ${geos_ALL_HEADERS})
-  add_library(geos-static STATIC ${geos_SOURCES} ${geos_ALL_HEADERS})
+#  add_library(geos-static STATIC ${geos_SOURCES} ${geos_ALL_HEADERS})
 
   set_target_properties(geos
     PROPERTIES
@@ -70,11 +70,11 @@
     VERSION ${VERSION}
     CLEAN_DIRECT_OUTPUT 1)
 
-  set_target_properties(geos-static
-    PROPERTIES
-    OUTPUT_NAME "geos"
-    PREFIX "lib"
-    CLEAN_DIRECT_OUTPUT 1)
+  # set_target_properties(geos-static
+  #   PROPERTIES
+  #   OUTPUT_NAME "geos"
+  #   PREFIX "lib"
+  #   CLEAN_DIRECT_OUTPUT 1)
 
 endif()
 
@@ -90,7 +90,7 @@
 #################################################################################
 
 if(NOT GEOS_ENABLE_MACOSX_FRAMEWORK)
-  install(TARGETS geos geos-static
+  install(TARGETS geos
     RUNTIME DESTINATION bin
     LIBRARY DESTINATION lib
     ARCHIVE DESTINATION lib)
